src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -292,16 +302,20 @@ public GerritTrigger(List<GerritProject> gerritProjects) {
      */
     @Deprecated
     public GerritTrigger(List<GerritProject> gerritProjects, SkipVote skipVote, Integer gerritBuildStartedVerifiedValue,
```

Comment:
Do not change signature of old deprecated constructors. They are only left here to keep backwards binary compatibility.
Also these values you are adding are not mandatory so they should not be in any constructor.